src/com/rcv/Main.java

```diff
@@ -154,9 +168,13 @@ static String executeTabulation(ElectionConfig config) {
   // param: config object containing CVR file paths to parse
   // returns: list of all CastVoteRecord objects parsed from CVR files
   private static List<CastVoteRecord> parseCastVoteRecords(ElectionConfig config) throws Exception {
+    if (config.rawConfig.cvrFileSources == null || config.rawConfig.cvrFileSources.isEmpty()) {
+      Logger.severe("Config doesn't contain any CVR input files.");
+    }
+
     // castVoteRecords will contain all cast vote records parsed by the reader
     List<CastVoteRecord> castVoteRecords = new ArrayList<>();
-    // at each iteration of the following loop we add records from another source file
+    // At each iteration of the following loop, we add records from another source file.
```

Comment:
bit of a deviation in style here, we don't seem to capitalize or punctuate inline comments anywhere else.

Reply:
True. I changed this one because I was having a little trouble parsing the comment. I'm OK with being lax on such things in general, but I think it's fine to occasionally make comments more clear when one of us is sufficiently motivated.